supervisor/utils/blockbuster.py

```diff
@@ -1,24 +1,28 @@
 """Activate and deactivate blockbuster for finding blocking I/O."""
```

Comment:
GH doesn't let me apply suggestion over the whole file, but this should be the whole module content:
```suggestion
"""Activate and deactivate blockbuster for finding blocking I/O."""

import logging

from blockbuster import BlockBuster as _BlockBuster

_LOGGER: logging.Logger = logging.getLogger(__name__)


class BlockBuster:
    _instance: _BlockBuster | None = None

    @classmethod
    def is_enabled(cls):
        """Return true if blockbuster detection is enabled."""
        if cls._instance is None:
            return False
        for _, fn in cls._instance.functions.items():
            return fn.activated
        return False

    @classmethod
    def activate(cls):
        """Activate blockbuster detection."""
        _LOGGER.info("Activating BlockBuster blocking I/O detection")
        if cls._instance is None:
            cls._instance = _BlockBuster()
        cls._instance.activate()

    @classmethod
    def deactivate(cls):
        """Deactivate blockbuster detection."""
        _LOGGER.info("Deactivating BlockBuster blocking I/O detection")
        if cls._instance:
            cls._instance.deactivate()
```

Reply:
Yeah seems cleaner. I'll rename it to `BlockBusterManager` to avoid the naming issue.